media/css/pebbles/components/_global-nav.scss

```diff
@@ -0,0 +1,594 @@
+        &:link,
+        &:visited {
+            @include font-size(14px);
+            background: inherit;
+            border-radius: 0;
+            border: 1px solid $color-firefox-light-orange;
+            color: $color-firefox-light-orange;
+            display: inline-block;
+            padding: 13px 10px;
+            text-decoration: none;
+            transition: color 0.1s ease-in-out, background-color 0.1s ease-in-out;
+        }
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-color: $color-firefox-light-orange;
+            color: #fff;
+            transition: color 0.1s ease-in-out, background-color 0.1s ease-in-out;
+        }
+    }
+
+    @media #{$mq-nav-desktop} {
+        margin-right: 42px;
+
+        .download-link {
+            &:link,
+            &:visited {
+                padding: 13px 20px;
+            }
+        }
+    }
+
+    .fx-privacy-link {
+        display: none;
+    }
+}
+
+.other #global-nav-download-firefox,
+.oldwin #global-nav-download-firefox,
+.oldmac #global-nav-download-firefox {
+    display: none;
+}
+
+/* -------------------------------------------------------------------------- */
+// Side drawer state rules.
+
+// Default (non-animated) accordion for old browsers.
+.moz-global-nav-drawer {
+
+    .detail {
+        display: none;
+    }
+
+    .summary.selected + .detail {
+        display: block;
+    }
+}
+
+// Animated accordion for newer browsers.
+@supports (transition: max-height .6s ease-in-out, visibility 0s 0s) {
+
+    .moz-global-nav-drawer {
+
+        .detail {
+            display: block;
+            max-height: 0;
+            overflow: hidden;
+            visibility: hidden;
+            transition: max-height .3s ease-in-out, visibility 0s .8s;
+        }
+
+        .summary.selected + .detail {
+            max-height: 1000px;
+            visibility: visible;
+            transition: max-height .6s ease-in-out, visibility 0s 0s;
+        }
+    }
+}
+
+// Default (non-animated) drawer for old browsers.
+body {
+    position: relative;
+    overflow-x: hidden;
+}
+
+html.moz-nav-open {
+    body {
+        left: 320px;
+    }
+}
+
+// Animated drawer for newer browsers.
+@supports (transform: translateX(320px)) {
+    body {
+        position: static;
+        transition: transform .25s ease-in-out;
+    }
+
+    html.moz-nav-open {
+        body {
+            left: 0;
+            transition: transform .25s ease-in-out;
+            transform: translateX(320px);
+        }
+
+        .moz-global-nav-drawer {
+            transition: visibility 0s 0s;
+            visibility: visible;
+        }
+    }
+
+    .moz-global-nav-drawer {
+        left: 0;
+        transform: translateX(-320px);
+        transition: visibility 0s .4s;
+        visibility: hidden;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Side drawer promo thumbnails.
+
+.moz-global-nav-drawer {
+
+    .nav-promo {
```

Comment:
I was in such a rush to get these in I completely forgot to do this, thanks for the reminder 👍